### ISSUE TITLE
Can not always find configurations for application when using flavors

### ISSUE THREAD

**Issue by btlogy** [stdevteam/fastlane-plugin-find_firebase_app_id#3]
I might have hit a bug/feature with this line:
https://github.com/STDevTM/fastlane-plugin-find_firebase_app_id/blob/5732c6790c88139c1d30382133ad0988545d67c2/lib/fastlane/plugin/find_firebase_app_id/actions/find_firebase_app_id_action.rb#L46

If the project owner split google-services.json files in sub-directories per flavor, like this:
- ./app/src/flavor1/google-services.json -> contains id for com.example.app.flavor1 only
- ./app/src/flavor2/google-services.json -> contains id for com.example.app.flavor2 only

Fastlane action `find_firebase_app_id` called for 'com.example.app.flavor2' will parse only the first one and ends up with this message:
`google-services.json file doesn't contain configurations for com.example.app.flavor2 application.`

The obvious work-around is to make sure the first google-services.json file found in alphabetic order will at least hold the flavor id of the flavor you want to send to Firebase...

**Comment by btlogy** [comment]
Of course, a better solution would be to parse all files rather than the first one only. And retain the first match for the package name requested.
But I'm sorry to say I'm not going to spend time to PR this kind of solution. :-/

**Comment by hamtiko** [comment]
@btlogy thanks for reporting.

Actually when you have more than one app in firebase you can use one `google-service.json` file for both apps as Firebase includes all android apps credentials in same file (only for Android).

I don't think that it is worth to support multiple file in this case.
